src/test/java/core/basesyntax/ReportServiceTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportServiceTest {
+    private static FruitDao dao;
+    private static ReportService reportService;
+
+    @BeforeClass
+    public static void beforeAll() {
+        dao = new FruitDaoImpl();
+        reportService = new ReportServiceImpl(dao);

```

Comment:
```suggestion
        reportService = new ReportServiceImpl(new FruitDaoImp());

```
You need dao only to create reportService. I think this way to write is better